### DISCUSSION TITLE
코테 깃헙 이용법

### DISCUSSION THREAD

**Discussion by Dangtae** [happyCoVid-19/PS#7] [Announcements]
@8eomio @limhongcheol @KOKOLOCOKES @Dangtae @Solsol1014 @minrongtic 

![폴더정리](https://github.com/user-attachments/assets/43b11fd0-9899-4d62-893d-ca8945972200)

각자 `Branch` 에 들어가면 이렇게 폴더들 있을텐데

- 해당주차폴더에 들어가기
- 본인이름 폴더 생성
- 코딩테스트 문제 파일 넣기

이렇게 push해주면 될것같아~ 
폴더안에다가 이름폴더 만드는 것 꼭 유념하기!

**Comment by Solsol1014** [comment]
넹면~

**Comment by minrongtic** [comment]
당면~
